README.md

```diff
@@ -0,0 +1,15 @@
+# Akka Platform Operator Charts
+
+This repository contains Helm charts for the [Akka Platform Operator](https://developer.lightbend.com/docs/akka-platform-guide/deployment/).
+
+## Maintenance
```

Comment:
cc. @sebastian-alfers @JustinPihony 

Maintenance as Console chart (this repo have the same purpose):
https://github.com/lightbend/console-charts#maintenance